embeddings/embedding/static/magnitude.py

```diff
@@ -22,12 +22,12 @@ class MagnitudeConfig:
     model_path: str
 
 
-class MagnitudeEmbedding(Embedding[List[str], ndarray]):
+class MagnitudeEmbedding(Embedding[List[str], nptyping.NDArray[Any]]):
```

Comment:
Is using `Any` everywhere in `nptyping.NDArray` good practice or we will refactor that at later stage?

Reply:
so we should be exact, but lets do it in next step